charon/utils/strings.py

```diff
@@ -2,3 +2,8 @@ def remove_prefix(string: str, prefix: str) -> str:
     if string and prefix and string.startswith(prefix):
         return string[len(prefix):]
     return string
+
+def remove_sufix(string: str, sufix: str) -> str:
```

Comment:
Um... where is this remove_suffix func used? I don't see it's called anywhere.

Reply:
I’ll remove this